src/rendering/format.jl

```diff
@@ -0,0 +1,48 @@
+# TODO:
+# - 1. do assertions for definition mandatory fields in `@define_format` macro
+# - 2. implement fallback format/rendering functions in format.jl
+# - 3. export this as public API
+
+
```

Comment:
let's remove or modify these comments :P

Reply:
> > if you want you can create another PR on top of this branch. If you set base branch to this PR, the diff will be taken from here.
> 
> Ah, ok. I've never done that before. I'll give it a try

So I created a new PR but since *this* PR lives on a branch on my personal fork,
the new PR lives in my own fork entirely...
https://github.com/JonasIsensee/Weave.jl/pull/1